.circleci/config.yml

```diff
@@ -24,14 +24,10 @@ jobs:
           command: |
             sudo apt-get update
             sudo apt-get install -y libpng-dev default-mysql-client
```

Comment:
This probably needs a `sudo apt-get update && ` to match https://github.com/palantirnet/drupal-skeleton/pull/119/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R25

Reply:
Well, it already has one right above, but I moved them to the same line to stay consistent with other config.yml files.